Tag Compat v0.9.0 [https://github.com/JuliaLang/Compat.jl]

andreasnoack opened the pull request.

ararslan commented:
I guess you'd have to bump the Julia requirement to 0.4 in order for this to pass Travis here.

andreasnoack commented:
Will wait for https://github.com/JuliaLang/Compat.jl/pull/279

andreasnoack commented:
@tkelman I've updated this PR to a 0.9.0 tag that includes the commit that drops support for 0.3. I accidentally pushed to `JuliaLang` and even though I deleted the branch immediately, Travis seems to be confused. Are you able to merge anyway or should I open a new PR?
